src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -1985,7 +1985,7 @@ private static string EscapeFunctionNameForRemoteHelp(string name)
             StringBuilder result = new(name.Length);
             foreach (char c in name)
             {
-                if (("\"'`$".IndexOf(c) == (-1)) &&
+                if ((!"\"'`$".Contains(c)) &&
```

Comment:
Maybe:
```suggestion
                if (!("\"'`$".Contains(c)) &&
```

Reply:
The parenthesis are unnecessary and would be a rule violation of [IDE0047](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0047) and [SA1119](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1119.md).

Reply:
Why not remove the parenthesis at all?
```suggestion
                if (!"\"'`$".Contains(c) &&
```

Reply:
I try to submit code quality PRs without style changes, but as you asked, I've pushed some more commits to address these issues.